components/landing/HeroBanner.vue

```diff
@@ -36,6 +39,36 @@
   </section>
 </template>
 
+<script lang="ts" setup>
+import farcasterLiveMintConfig from '@/script/farcasterLiveMint.json'
+import { $fetch } from 'ofetch'
```

Comment:
auto import, `$fetch` already built-in https://nuxt.com/docs/api/utils/dollarfetch

Reply:
updated